protobuf-format.md

```diff
@@ -138,12 +137,40 @@ Transports that support content identification MUST use the following designatio
    application/cloudevents+protobuf
 ```
 
-## 5. Examples
+## 5. Batch Format
+
+Batch format allows for a set of CloudEvents to be represented, no relationship
+between those events is implied.
+
+Although the _protobuf batch format_ builds on the _protobuf format_ it is considered
+seperate, that is to say that support of _protobuf format_ does not indicate support
+of the batch representation. The batch format MUST only be used where supported.
```

Comment:
I'm not sure how to interpret that MUST. Is that a statement for the sender or receiver? How does the sender know?

Reply:
There's similar language in the JSON format specification .. the essence is that just because somebody claims to support _cloudevents+json_ does NOT imply that they also support _cloudevents-batch+json_.

I can tweak the language a bit more if desired, or copy-paste form the JSON spec ;-)